run_ci.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+sudo apt update
+sudo apt install -y python3 python3-venv python3-pip
+
+VENV_DIR="ci_venv"
+
+python3 -m venv "$VENV_DIR"
+source "$VENV_DIR/bin/activate"
+
+python3 -m pip install --upgrade pip
+pip install pylint
+pip install black
+
+echo ===== SETUP AND INSTALLATION OF PACKAGES COMPLETE =====
```

Comment:
We can add a conditional to check if `ci_venv` is already created. If it is, then we don't have to run all the installation again since presumably it's already set up.